### ISSUE TITLE
`dynamic_scheduler` ServiceWaitingForManualInterventionError - developers should be notified

### ISSUE THREAD

**Issue by matusdrobuliak66** [ITISFoundation/osparc-simcore#6801]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Planned changes

When this is logged: 
```
servicelib.rabbitmq.rpc_interfaces.dynamic_scheduler.errors.ServiceWaitingForManualInterventionError: Service <uuid> waiting for manual intervention
```
Developers should be notified in some way; otherwise, we cannot know that action needs to be taken.

**Comment by GitHK** [comment]
Creator. This is a thing because we have a 0 data-loss policy in place. What should be done is find a way to act upon this message more quickly.

**Comment by matusdrobuliak66** [comment]
> Creator. This is a thing because we have a 0 data-loss policy in place. What should be done is find a way to act upon this message more quickly.

Yes, this issue is exactly for that **developers should be notified**
